baseline.md

```diff
@@ -1,68 +1,3 @@
-# The Simeon Baseline: A comprehensive set of best practice Microsoft 365 configurations.
```

Comment:
the top part here was removed because the Build-BaselineMarkdown script was not run from the docs repo - the working directory when running the script should be the docs repo. Revert these changes before running again (because it won't put it back now that it's been removed)